src/app/group/[groupId]/setting/page.tsx

```diff
@@ -1,16 +1,23 @@
+import { getGroupInfo, getGroupMemberList } from "@/api/group";
 import Sidebar from "@/common/component/Sidebar";
-import { tmpMemberListData } from "@/shared/util/example";
 import { sidebarWrapper } from "@/styles/shared.css";
 import MemberList from "@/view/group/setting/MemberList";
 import SettingSidebar from "@/view/group/setting/SettingSidebar";
 
-const GroupSettingPage = () => {
+const GroupSettingPage = async ({
+  params,
+}: { params: { groupId: string } }) => {
+  const groupData = getGroupInfo(+params.groupId);
+  const memberData = getGroupMemberList(+params.groupId);
```

Comment:
```suggestion
  params: {groupId},
}: { params: { groupId: string } }) => {
  const groupData = getGroupInfo(+groupId);
  const memberData = getGroupMemberList(+groupId);
```

이렇게 바꿔주시면 좋을 것 같아요 참조가 여러번 발생하네요